01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,6 @@
+for (let i = 1; i <= 20; i++) {
+    let output = '';
+    if (i % 3 === 0) output += 'Fizz';
+    if (i % 5 === 0) output += 'Buzz';
+    console.log(output || i);
```

Comment:
(修正は必要ないのですが) Ruby と違って、JavaScript には false とみなされる値 (falsy な値と言います) がたくさんあります。`||` 演算子を使うときには、意図せずに false と判定される場合があることに注意してください。この場合であれば、`output` が空文字列であることを厳密等価演算子を使ってチェックしておくほうが確実です。https://developer.mozilla.org/ja/docs/Glossary/Falsy

```suggestion
    console.log(output === '' ? output : i);
```